### PR TITLE
bugfix: rgx_split inital tag type

### PR DESCRIPTION
**Pull request by techofer**

### CODE REVIEW

File: spannerlib/span.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"""Span class and how to interface it with pandas"""
```
Comment (DeanLight) [inline spannerlib/span.py:1]: how did you change this without changing the notebooks?

**Review by DeanLight — CHANGES_REQUESTED** [review]
how?!

File: spannerlib/span.py (hunk 1 above)
Comment (techofer) [reply spannerlib/span.py:1]: I think the nbdev_export did that, maybe nbdev version thing?

Comment (techofer) [reply spannerlib/span.py:1]: yes

**Pull request merged**